<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parts</class>
 <widget class="QFrame" name="Parts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Part</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QcjDataFrame" name="dataFrame">
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="xml_definition" stdset="0">
         <string>form_definition</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QcjDataForm" name="dataForm">
              <property name="toolTip">
               <string/>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;This form for entering parts information into inventory. It's xml form name is &lt;em&gt;parts_form&lt;/em&gt;&lt;/html&gt;</string>
              </property>
              <property name="xml_definition" stdset="0">
               <string>parts_form</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="_3">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="margin">
                <number>9</number>
               </property>
               <item>
                <widget class="QPushButton" name="newBtn">
                 <property name="text">
                  <string>New Part</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="saveBtn">
                 <property name="text">
                  <string>Save Part</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteBtn">
                 <property name="text">
                  <string>Delete Part</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="searchBtn">
                 <property name="text">
                  <string>Search</string>
                 </property>
                 <property name="default">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearBtn">
                 <property name="toolTip">
                  <string>&lt;html&gt;Clicking this button clears the form&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Clear Form</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QcjDataTable" name="dataTable">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;This table is for selecting a part to modify from inventory. It's xml form name is &lt;em&gt;parts_table&lt;/em&gt;&lt;/html&gt;</string>
           </property>
           <property name="xml_definition" stdset="0">
            <string>parts_table</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QcjDataForm</class>
   <extends>QFrame</extends>
   <header>QcjData/QcjDataForm.h</header>
  </customwidget>
  <customwidget>
   <class>QcjDataFrame</class>
   <extends>QFrame</extends>
   <header>QcjData/QcjDataFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QcjDataTable</class>
   <extends>QFrame</extends>
   <header>QcjData/QcjDataTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
